packages/contracts/contracts/voting/addresslist/AddresslistVoting.sol

```diff
@@ -127,26 +122,31 @@ contract AddresslistVoting is MajorityVotingBase {
         // calculate start and end time for the vote
         uint64 currentTimestamp = getTimestamp64();
 
-        if (_startDate == 0) _startDate = currentTimestamp;
-        if (_endDate == 0) _endDate = _startDate + minDuration;
+        if (_startDate == 0) {
+            _startDate = currentTimestamp;
+        }
+        if (_endDate == 0) {
+            _endDate = _startDate + minDuration;
+        }
 
-        if (_endDate - _startDate < minDuration || _startDate < currentTimestamp)
+        if (_endDate - _startDate < minDuration || _startDate < currentTimestamp) {
             revert VotingPeriodInvalid({
                 current: currentTimestamp,
                 start: _startDate,
                 end: _endDate,
                 minDuration: minDuration
             });
+        }
 
         proposalId = proposalCount++;
 
-        // create a vote.
+        // Create the proposal
         Proposal storage proposal_ = proposals[proposalId];
         proposal_.startDate = _startDate;
         proposal_.endDate = _endDate;
         proposal_.snapshotBlock = snapshotBlock;
-        proposal_.relativeSupportThresholdPct = relativeSupportThresholdPct;
-        proposal_.totalSupportThresholdPct = totalSupportThresholdPct;
+        proposal_.supportThreshold = supportThreshold;
+        proposal_.participationThreshold = participationThreshold;
```

Comment:
If we agreed that `min` referred to `>=` and `threshold` referred to `>`, shouldn't the second parameter be named like `minParticipation`?

Reply:
Changed `participationThreshold` to `minParticipation` in aa72391bb842e8672bc30a8a77a6ffb7f18a7c17.